drivers/ukconsole/ns16550/ns16550.c

```diff
@@ -310,7 +310,7 @@ static int early_init(struct ukplat_bootinfo *bi)
 		return rc;
 	}
 
-	uk_console_init(&earlycon.dev, "NS16550", &ns16550_ops,
+	uk_console_init(&earlycon.dev, "ns16550", &ns16550_ops,
```

Comment:
Actually the device name is `NS16550` (with uppercase): https://archive.org/details/bitsavers_nationaldamunicationsElementsDataBook_19316911

Reply:
I'll change it back.

Reply:
Looks like this one hasn't been fixed, unresolving.

Reply:
There was probably a mistake happening during push. It is back in now.